CIP-0027.md

```diff
@@ -0,0 +1,64 @@
+
+
+## Simple Summary
+
+A community standard for royalties&#39; functionality, that does not require smart contracts to implement.
+
+## Abstract
+
+This proposed standard will allow for uniform royalties&#39; distributions across the secondary market space. It is easy to implement using metadata only, and does not require a smart contract.  However, it is scalable to allow for the usage of a downstream smart contract, as needed by the asset creator. It has been developed with input from Artano, BuffyBot, CNFT.io, Digital Syndicate, Fencemaker, MADinArt, NFT-Maker.io, Hydrun, Tokhun, and many more.
+
+## Motivation
+
+There is a significant interest within the Cardano Community for an implementation of royalties distribution when a Cardano Asset is resold on the secondary market. It has become a common theme to see and hear statements that the only thing stopping artists from adopting Cardano, is that they are waiting for an implementation of royalties.  At the present time, smart contracts do not create a simple mechanism to implement royalties.  By developing a community standard, we can resolve the immediate need for royalties, and create a path forward for a potential future iteration of smart contracts.
+
+## Specification
+
+A new tag of 777 is proposed for this implementation.  The community guidelines have been agreed as follows:
```

Comment:
Please add the tag to the registry as well instead of just mentioning the tag in your CIP

Reply:
Will do